### PR TITLE
Add .replace(/\\./g, "/") to gatsby.node 

### PR DESCRIPTION
**Pull request by corwintines**
## Description

- add .replace(/\\./g, "/") to `const dirname = path.resolve('./)`

### CODE REVIEW

**Comment by corwintines** [comment]
@TylerAPfledderer do you mind confirming this fixes the tempSplit issue for you 🙏

File: gatsby-node.ts [hunk 1]
```diff
@@ -38,7 +38,8 @@ const commonRedirectProps = {
  * @returns boolean for if file is outdated or not
  */
 const checkIsMdxOutdated = (filePath: string): boolean => {
-  const dirname = path.resolve("./")
+  // .replace(/\\./g, "/") to replace \ in windows paths ex: C:\\folder\\myfile.txt becomes C:/folder/myfile.txt
+  const dirname = path.resolve("./").replace(/\\./g, "/")
```
Comment (TylerAPfledderer) [inline gatsby-node.ts:42]: It actually should not have the dot.  It is not used for the problem we are trying to solve, and it's deleting the first letter after each slash.
```suggestion
  // .replace(/\\/g, "/") to replace \ in windows paths ex: C:\\folder\\myfile.txt becomes C:/folder/myfile.txt
  const dirname = path.resolve("./").replace(/\\/g, "/")
```

**Review by TylerAPfledderer — CHANGES_REQUESTED** [review]
@corwintines I gave you an error in this regex I suggested.

File: gatsby-node.ts (hunk 1 above)
Comment (corwintines) [reply gatsby-node.ts:42]: Woops, mistyped it. Thanks!

**Review by nhsz — APPROVED** [review]
LGTM 🪟

File: gatsby-node.ts [hunk 2]
```diff
@@ -38,7 +38,8 @@ const commonRedirectProps = {
  * @returns boolean for if file is outdated or not
  */
 const checkIsMdxOutdated = (filePath: string): boolean => {
-  const dirname = path.resolve("./")
+  // .replace(/\\/g, "/") to replace \ in windows paths ex: C:\\folder\\myfile.txt becomes C:/folder/myfile.txt
+  const dirname = path.resolve("./").replace(/\\/g, "/")
```
Comment (pettinarip) [inline gatsby-node.ts:42]: There are a few other places were we use `path.resolve`. Wondering why we wouldn't have this problem in those cases.

Also, `path.resolve` takes care of this problem https://nodejs.org/api/path.html#windows-vs-posix

I'm trying to understand why we need to do this and why this is not breaking in other places.

**Pull request merged**